doc/users/index.rst

```diff
@@ -4,7 +4,7 @@
 
 
 ###########
-Users guide
+User guide
 ###########
```

Comment:
I would have expected the “#” lines need to be the same length as the title, but the doc build doesn’t seem to care in this case 🤔🤷‍♀️

Reply:
I think they need to be _at least_ as long, but we do prefer them to match.